src/audio/Kconfig

```diff
@@ -24,7 +24,11 @@ config COMP_FIR
 	bool "FIR component"
 	default y
 	help
-	  Select for FIR component
+	  Select for FIR component. Note that FIR equalizer is recommended
+	  to be used in Xtensa DSP architecture when built with xt-xcc
+	  compiler only. As xtensa gcc build the max. filter tap count is
+	  severely restricted due to slow multiply-and-accumulate performance
+	  in generic C version.
```

Comment:
I would re-write this to be more generic. i.e. "FIR performance can differ between DSP ISA and compiler used. FIR will perform best when MAC instructions are available in ISA and are used by the compiler. i.e. xt-xcc on xtensa will generate MAC instructions but GCC on xtensa wont. Filter tap cound can be severly restricted to reduce FIR cycles and FIR performance for DSP/compilers with no MAC support".

Reply:
Thanks, I'll replace. It's much better than my version.

Reply:
Done!